doc/quickstart/find_data.rst

```diff
@@ -45,6 +48,39 @@ From the ESMValTool user perspective the number of data input parameters is
 optimized to allow for ease of use. We detail this procedure in the next
 section.
 
+Native model data
+---------------------------------------------------------
+Support for native model data is quite easy using basic
```

Comment:
```suggestion
Support for native model data that is not formatted according to a CMIP data request is quite easy using basic
```
would be a bit more specific. Some models just write their output in a way that already follow the CMIP data request, so no reformatting is needed at all.

Reply:
Changed